python/sdk/Makefile

```diff
@@ -4,7 +4,7 @@ setup:
 
 .PHONY: type_check
 type_check:
-	@pipenv run mypy --ignore-missing-imports --allow-untyped-globals  merlin
+	@pipenv run mypy --ignore-missing-imports --allow-untyped-globals --implicit-optional merlin
```

Comment:
what is the purpose of `implicit-optional`

Reply:
It's to fix issue in mypy due to the changes in its default behavior. Anyway, it's been addressed in https://github.com/gojek/merlin/pull/332